Kitodo-DataManagement/src/test/java/org/kitodo/data/database/persistence/AuthorityDaoIT.java

```diff
@@ -16,14 +16,27 @@
 import java.util.ArrayList;
 import java.util.List;
 
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.rules.ExpectedException;
+import org.kitodo.MockIndex;
 import org.kitodo.data.database.beans.Authority;
 import org.kitodo.data.database.exceptions.DAOException;
 
 public class AuthorityDaoIT {
 
+    @BeforeClass
+    public static void setUp() throws Exception {
+        MockIndex.startNode();
+    }
+
+    @AfterClass
+    public static void tearDown() throws Exception {
+        MockIndex.stopNode();
+    }
+
```

Comment:
You are adding similar code to "integration test" classes. Would it not be better to use a class for this as all of them are needing this changes and extending the integration classes from this base integration class?

Reply:
Understandable and nothing for now.